actions/consul/verify_installation_consul/action.yaml

```diff
@@ -0,0 +1,26 @@
+name: 'Verify Consul Installation Action'
+description: 'A GitHub Action to verify tests, get logs from pods'
+inputs:
+  namespace:
+    description: 'namespace where service is installed'
+    required: true
+    default: consul
+  max_attempts:
+    description: 'maximum number of attempts to verify that pod status is successful'
+    required: true
+    default: 40
+  timeout:
+    description: 'timeout between attempts'
+    required: true
+    default: '10s'
+
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Run helm_install action
+      uses: Netcracker/qubership-test-pipelines/actions/shared/verify_installation@update_actions
```

Comment:
Could you use relative path without `@update_actions`?

Reply:
fixed